app/controllers/schools_controller.rb

```diff
@@ -28,8 +28,8 @@ def csv
   end
 
   def absence_dashboard_data
-    student_absence_data = students_for_dashboard(@school)
-      .includes([homeroom: :educator], :dashboard_absences, :event_notes)
+    student_absence_data = active_students(@school)
```

Comment:
I renamed this method because it added an extra step to reading/writing code in this class. The first time I saw the `students_for_dashboard` method, it made me wonder, "ooooh, I wonder what queries we're running to get the right students for the dashboard!" and went and looked at the method. But it turned out we were just querying for active students. I changed the method name to be a little more upfront about what it's doing.